x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SnapshotRetentionTask.java

```diff
@@ -109,6 +119,7 @@ public void triggered(SchedulerEngine.Event event) {
                     slmStats.retentionFailed();
                     updateStateWithStats(slmStats);
                 } finally {
+                    logger.info("SLM retention snapshot cleanup task completed with error");
```

Comment:
is this needed? the `logger.error` above will print a message already

Reply:
I wanted to have something that signaled right when the `running.set(false)` finished. In this case, the `logger.error` above could print, but then `updateStateWithStats` could hang or not complete and we would have no indication.

Reply:
Gotcha, thanks Lee